spec/lib/api/v3/work_packages/schema/work_package_schema_spec.rb

```diff
@@ -92,9 +79,22 @@
           expect(subject.assignable_statuses_for(user)).to eql(status_result)
         end
       end
+    end
+
+    describe '#available_custom_fields' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.